README.md

```diff
@@ -75,6 +98,58 @@ Finally, add the bundle as a script tag in your page, and initialize the player
 </div>
 ```
 
+## Configuration:
+
+### Kava Plugin Properties
+
+- `serviceUrl` optional -  The Kaltura API server. default is kalturas default analytics url
+- `requestMethod`  optional - The http method to be used for sending the beacons, GET or POST, the default is GET.
+- `viewEventCountdown` optional - The interval in seconds that VIEW event will be sent. default is every 10 seconds
+- `resetSessionCountdown` optional - The interval in seconds that Kava session will be reset. default is 30 seconds
+- `dvrThreshold` optional - Threshold in seconds from the live edge. default is 120 seconds
+- `userId`  optional - custom user id .
+- `applicationVersion` optional -  Used to send the application version from which the user is playing the entry.
+- `customVar1` optional - Custom object field.
+- `customVar2` optional -  Custom object field.
+- `customVar3` optional -  Custom object field.
+- `persistentSessionId` optional -  UUID for anonymous users.
+- `playbackContext`  optional - Used to send the id of the category from which the user is playing the entry. (usually populated by the player)
+- `tamperAnalyticsHandler`  Optional handler to implement. Can be used to manipulate the model data before analytics event sent, or to cancel a certain analytics request.
+
+
+### Example:
+
+```javascript
+// Default config
+"kava": {
+ serviceUrl: '//analytics.kaltura.com/api_v3/index.php',
```

Comment:
why is it commented ?
if this is a sample url just write serviceUrl: [serviceUrl] or something